### PR TITLE
Implement helpful question

### PR DESCRIPTION
**Pull request by brynrmrzz**
https://trello.com/c/A44XQbTy/127-answer-post-request

voting yes will now sent a put request to the server, it will also disable the button in using localStorage.

### CODE REVIEW

File: src/components/questions/QuestionsAnswers.jsx [hunk 1]
```diff
@@ -74,19 +74,28 @@ class QuestionsAnswers extends React.Component {
...
-      this.setState({ questionsList: newState });
-      button.disabled = true;
+    const { questionsList, view } = this.state;
+    const { productId } = this.props;
+    const targetQuestionId = questionToUpdate.question_id;
+    const button = document.querySelector(`#vote-helpful-question-${targetQuestionId}`);
+
+    if (!window.localStorage[targetQuestionId]) {
+      putHelpfulQuestion(targetQuestionId)
+      .then(()=> {
```
Comment (kjniemela) [inline src/components/questions/QuestionsAnswers.jsx:85]: Possible indentation error?

File: src/styles.scss [hunk 2]
```diff
@@ -1003,13 +1003,7 @@ body {
...
   }
 }
-/* Reviews ---------------------(end) */
 
-/* Questions ---------------------(start) */
-
-/* Questions ---------------------(end) */
-
-/* Shared ---------------------(start) */
 .starInputContainer {
```
Comment (kjniemela) [inline src/styles.scss:1007]: Did you mean to delete the Shared marker?

File: src/App.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ class App extends React.Component {
     super(props);
     this.state = {
       productId: null,
-      outfitItems: []
+      outfitItems: [],
```
Comment (anerolabs) [inline src/App.jsx]: This will be over-written in the cache refactor, so maybe leave out the added comma to avoid a conflict?

**Review by anerolabs — COMMENTED** [review]
Digging the utils file, it really does help keep your components clean!

File: src/styles.scss (hunk 2 above)
Comment (brynrmrzz) [reply src/styles.scss:1007]: No, this was a merge conflict tat I just accepted incoming changes. Not intentional sorry!

File: src/components/questions/QuestionsAnswers.jsx (hunk 1 above)
Comment (brynrmrzz) [reply src/components/questions/QuestionsAnswers.jsx:85]: thank you!

**Pull request merged**